packages/kit/src/runtime/client/client.js

```diff
@@ -64,29 +64,37 @@ const scroll_positions = storage.get(SCROLL_KEY) ?? {};
  */
 const snapshots = storage.get(SNAPSHOT_KEY) ?? {};
 
-const original_push_state = BROWSER ? history.pushState : () => {};
-const original_replace_state = BROWSER ? history.replaceState : () => {};
-
 if (DEV && BROWSER) {
 	let warned = false;
 
 	const warn = () => {
 		if (warned) return;
+
+		// Avoid stashing away the original history methods which prevents monkeypatching by other libs,
+		// instead inspect the stack trace to see if we're being called from within SvelteKit.
```

Comment:
I might reword this slightly in case "stashing away" isn't clear
```suggestion
		// Rather than saving a pointer to the original history methods, which would prevent monkeypatching by other libs,
		// inspect the stack trace to see if we're being called from within SvelteKit.
```